irohad/ametsuchi/impl/flat_file_block_query.cpp

```diff
@@ -0,0 +1,85 @@
+/**
+ * Copyright Soramitsu Co., Ltd. 2017 All Rights Reserved.
+ * http://soramitsu.co.jp
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "ametsuchi/impl/flat_file_block_query.hpp"
+
+#include "model/converters/json_common.hpp"
+
+namespace iroha {
+  namespace ametsuchi {
+    FlatFileBlockQuery::FlatFileBlockQuery(FlatFile &block_store)
+        : block_store_(block_store) {}
```

Comment:
Class 'FlatFileBlockQuery' has a constructor with 1 argument that is not explicit.